Frameworks/UIKit/UIButton.mm

```diff
@@ -184,7 +184,7 @@ - (void)_initUIButton {
     _contentVerticalAlignment = UIControlContentVerticalAlignmentCenter;
     _contentHorizontalAlignment = UIControlContentHorizontalAlignmentCenter;
 
-    __block UIButton* weakSelf = self;
+    __weak UIButton* weakSelf = self;
```

Comment:
No.  UIView doesn't have ARC enabled, so you can't use __weak.

---
In reply to: [98295161](https://github.com/Microsoft/WinObjC/pull/1861#discussion_r98295161) [](ancestors = 98295161)